lib/logitech_receiver/hidpp10.py

```diff
@@ -116,6 +128,25 @@
     notifications=0x00,
     firmware=0xF1,
 )
+# Flags taken from https://drive.google.com/drive/folders/0BxbRzx7vEV7eWmgwazJ3NUFfQ28
```

Comment:
Hi,
This is the link to the folder, it might be handy to mention which file you used from this folder?

Reply:
Hi,
links updated.